pkg/config/application_test.go

```diff
@@ -210,11 +210,13 @@ func TestValidateEncryption(t *testing.T) {
 	testcases := []struct {
 		name             string
 		encryptedSecrets map[string]string
+		targets          []string
 		wantErr          bool
 	}{
 		{
 			name:             "valid",
 			encryptedSecrets: map[string]string{"password": "pw"},
+			targets:          []string{"secret.yaml"},
```

Comment:
[IMO] add `target` for the other patterns "invalid because key is empty", "invalid because value is empty" for the correct test case 👀